server/src/userManagement.mjs

```diff
@@ -124,10 +128,19 @@ function associateUserWithWss(userId, wss) {
   user2wss.set(''+userId, wss);
 }
 
+//Associate each user with many ws connection,taking ws as an array of objects
 function associateUserWithWs(userId, ws) {
-  user2ws.set(''+userId, ws);
+  let wsArray = [];
+  if(user2ws.has(''+userId)){
+    wsArray = user2ws.get(''+userId)
+    wsArray.push(ws);
+  } else {
+    wsArray.push(ws)
```

Comment:
These two lines are fine, but could be replaced by:
wsArray.push([ ws ]);

Reply:
wsArray.push([ws]),places array of ws inside an array,So when userManagement.getWsForUser is called,It is retrieving the values as [test] and validation steps are getting failed,because the expected value is "test''

Reply:
whoops... actually meant wsArray = [ ws ]; but what you have is fine.